Makefile.custom.mk

```diff
@@ -0,0 +1,24 @@
+# Generate kustomize patches and all helm charts
+.PHONY: generate
+generate:
```

Comment:
Summary of steps top generate Helm charts:
1. Download upstream manifests
2. Generate kustomize patches for webhooks (these change as upstream CRDs change, so we generate them)
3. Run `kustomize build` to generate Helm charts
4. Since all Helm charts ended up in a single output directory, we organize them per CAPI component (core, controlplane, bootstrap)
5. Break up CRD manifests into base file with multiple JSON patch files with OpenAPI schema for every API version